src/site/confluence/compatibility.confluence

```diff
@@ -1,10 +0,0 @@
-h1. API Compatibility
```

Comment:
These compatibility reports are left over. I think @tisonkun already has solution for javadoc. This sounds similar. It is a a good to have compatibility reports , but without ci in old site, these were dated anyway.

Reply:
Yeah. I think these reports should actually report for every PR and guard that we're in a healthy state instead of showing on the website - I highly suspect who will read these pages.